.github/workflows/integration-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: Integration tests
+name: Integration Tests
```

Comment:
This is to make it consistent with "Unit Tests" and look nicer on the badge

Reply:
I hadn't paid attention but this is a very specific integration test for an `(ollama + inference)` combo. Calling it "Integration Tests" isn't quite right. I am not sure we should upgrade this into a badge as yet.

Reply:
This is a versatile file for integration tests, designed to be extended for multiple providers and test scenarios.